app_dart/test/service/luci_test.dart

```diff
@@ -102,4 +102,34 @@ void main() {
     final List<Build> resultBuilds = await service.getBuildsForBuilderList(<LuciBuilder>[builder], repo: 'flutter');
     expect(resultBuilds, builds);
   });
+
+  test('luci getPartialBuildersList works correctly', () async {
```

Comment:
Would it be clearer to say `getPartialBuildersList handles non-uniform batches`?

Reply:
Updated.